<launch>

  <arg name="suction_control"	default="suction_control"/>
  <arg name="operation_mode"	default="operation_mode"/>

  <node name="suction_controller"
	pkg="aist_suction_controller" type="suction_controller.py"
	output="screen">
    <rosparam file="$(dirname)/../config/$(arg suction_control).yaml"/>
    <rosparam file="$(dirname)/../config/$(arg operation_mode).yaml"/>
  </node>

</launch>
